src/main/java/colby/Event.java

```diff
@@ -0,0 +1,27 @@
+package colby;
+
+/**
+ * Class that represents event tasks
+ * description refers to what the task is about
+ * start and end refer to the start time/date and end time/date of the event
+ */
+public class Event extends Task {
+    private final String start;
+    private final String end;
```

Comment:
I really like the naming conventions for your variables, they are short yet clearly indicates what they represent